pupil_src/shared_modules/accuracy_visualizer.py

```diff
@@ -386,6 +386,9 @@ def recalculate(self):
         )
 
         if not results.is_valid:
+            logger.info(
+                "Did not collect enough data to estimate gaze mapping accuracy."
+            )
```

Comment:
Please change to a warning in both cases and remove duplicated string definition.